server/main.py

```diff
@@ -565,55 +565,61 @@ def mcf_playground():
 
 
 #
-# STATIC PAGES FROM DATACOMMONS.ORG
+# STATIC PAGES
 #
 
-@app.route('/faq')
-def faq():
-  return flask.render_template('factcheck/faq.html')
 
+@app.route('/about')
+def about():
+  return flask.render_template('about.html')
 
-@app.route('/documentation')
-def documentation():
-  return flask.render_template('factcheck/documentation.html')
+
+@app.route('/faq')
+def faq():
+  return flask.render_template('faq.html')
 
 
 @app.route('/disclaimers')
 def disclaimers():
-  return flask.render_template('factcheck/disclaimers.html')
-
-
-@app.route('/getinvolved')
-def get_involved():
-  return flask.render_template('factcheck/getInvolved.html')
-
-
-@app.route('/colab')
-def colab():
-  return flask.render_template('factcheck/colab.html')
-
-
-@app.route('/data')
-def data():
-  return flask.render_template('factcheck/data.html')
+  return flask.render_template('disclaimers.html')
 
 
 @app.route('/datasets')
 def datasets():
-  return flask.render_template('factcheck/datasets.html')
+  return flask.render_template('datasets.html')
 
 
 @app.route('/special_announcement')
 def special_announcement_homepage():
   recent_blobs = list_blobs(_SA_FEED_BUCKET, _MAX_BLOBS)
   return flask.render_template(
-      'factcheck/special_announcement.html', recent_blobs=recent_blobs)
+      'special_announcement.html', recent_blobs=recent_blobs)
 
 
 @app.route('/special_announcement/faq')
 def special_announcement_faq():
   return flask.render_template(
-      'factcheck/special_announcement_faq.html')
+      'special_announcement_faq.html')
+
+
+#
```

Comment:
do you think it worth creating new module under routes for these? Eventually we can make all these routes out of main.py

Reply:
Moved to a new route. I agree with the goal, though would like to eventually enforce url_prefix for our blueprints so that it's easy to find the handlers in the future. One could argue that these pages which are at the root of the site (according to their url), are main pages. Maybe static.py isn't the best name for the module..